organizations/forms.py

```diff
@@ -97,6 +97,13 @@ def save(self, *args, **kwargs):
         """
         try:
             user = get_user_model().objects.get(email__iexact=self.cleaned_data['email'])
+            # Send a notification email to this user to inform them that they
+            # have been added to a new organization.
+            invitation_backend().send_notification(
```

Comment:
Since the exceptions caught are related only to line 99 I'd recommend moving this whole new block to _after_ the `except` blocks.